# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../postfix/postfix-no-postconf.patch.cross
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Do not run postconf when installing through postfix-install, because it
it might not be executable by the host when cross-compiled.

--- a/postfix-install	2014-03-07 18:24:06.661626473 +0100
+++ b/postfix-install	2014-03-07 18:29:09.408652360 +0100
@@ -205,11 +205,6 @@
     exit 1
 }
 
-test -x bin/postconf || {
-    echo $0: Error: no bin/postconf file. Did you forget to run \"make\"? 1>&2
-    exit 1
-}
-
 case `uname -s` in
 HP-UX*) FMT=cat;;
      *) FMT=fmt;;
@@ -368,7 +363,6 @@
 
 : ${install_root=/}
 : ${tempdir=`pwd`}
-: ${config_directory=`bin/postconf -c conf -h -d config_directory`}
 
 # Find out the location of installed configuration files.
 
@@ -438,8 +432,6 @@
 	case "$junk" in
 	"") eval unset $name;;
 	esac
-	eval : \${$name=\`bin/postconf -c $CONFIG_DIRECTORY -h $name\`} ||
-	    exit 1
     done
 }
 
@@ -451,7 +443,6 @@
     case "$junk" in
     "") eval unset $name;;
     esac
-    eval : \${$name=\`bin/postconf -c conf -d -h $name\`} || exit 1
 done
 
 # Override settings manually.
@@ -724,27 +715,6 @@
 
     done
 
-# Save the installation parameters to main.cf even when they haven't
-# changed from their current default. Defaults can change between
-# Postfix releases, and software should not suddenly be installed in
-# the wrong place when Postfix is being upgraded.
-
-bin/postconf -c $CONFIG_DIRECTORY -e \
-    "daemon_directory = $daemon_directory" \
-    "data_directory = $data_directory" \
-    "command_directory = $command_directory" \
-    "queue_directory = $queue_directory" \
-    "mail_owner = $mail_owner" \
-    "setgid_group = $setgid_group" \
-    "sendmail_path = $sendmail_path" \
-    "mailq_path = $mailq_path" \
-    "newaliases_path = $newaliases_path" \
-    "html_directory = $html_directory" \
-    "manpage_directory = $manpage_directory" \
-    "sample_directory = $sample_directory" \
-    "readme_directory = $readme_directory" \
-|| exit 1
-
 # If Postfix is being installed locally from source code, do the
 # post-install processing now.
 
